pkg/pillar/types/zedroutertypes.go

```diff
@@ -1807,6 +1807,7 @@ type UnderlayNetworkConfig struct {
 	Error   string
 	Network uuid.UUID // Points to a NetworkInstance.
 	ACLs    []ACE
+	VlanID  uint32
```

Comment:
Would make sense to comment that 0 means no vlan filtering in place.
Also, naming-wise should we call this AccessVlanID? I don't know what is more clear and correct to networking folks.

Reply:
will change to AccessVlanID

Reply:
done